pkg/spec/cluster.go

```diff
@@ -154,6 +165,11 @@ type PodPolicy struct {
 	// bootstrap the cluster (for example `--initial-cluster` flag).
 	// This field cannot be updated.
 	EtcdEnv []v1.EnvVar `json:"etcdEnv,omitempty"`
+
+	// PV represents a Persistent Volume resource.
+	// If defined new pods will use a persistent volume to store etcd data.
+	// TODO(sgotti) unimplemented
+	PV *PVSource `json:"pv,omitempty"`
```

Comment:
@sgotti 

one thing to note here, the backup pod also need to respect (or explicitly ignore this) this in the future. right now, it saves intermediate snapshot data onto its container fs.